Makefile

```diff
@@ -10,7 +9,7 @@ build-images: build-all-images
 .PHONY: build-images
 
 test:
-	OPERATOR_LOGGING_IMAGE_STREAM=$(OPERATOR_LOGGING_IMAGE_STREAM) ./hack/test-e2e.sh
+	LOGGING_VERSION=5.0 ./hack/test-e2e.sh
```

Comment:
Should that be `LOGGING_VERSION=${LOGGING_VERSION:-5.0}` too allow override?

Reply:
not here I don't believe.  This is an explicit change to 5.0 branch so I don't believe we would desire for it to be anything else